src/frontend.rs

```diff
@@ -594,40 +588,49 @@ impl AsyncComponent for App {
             glib::Propagation::Stop
         });
 
-        // TODO: Re-enable macOS once https://github.com/subspace/space-acres/issues/183 and/or
-        //  https://github.com/subspace/space-acres/issues/222 are resolved
-        let tray_icon = if cfg!(target_os = "macos") {
-            None
-        } else {
-            let tray_icon = TrayIconBuilder::new();
-            #[cfg(target_os = "linux")]
-            let tray_icon = tray_icon
-                .with_icon(Icon::from_png_bytes(ICON).expect("Statically correct image; qed"));
-            #[cfg(windows)]
-            let tray_icon = tray_icon
-                .with_icon(Icon::from_resource(1, None).expect("Tray icon is a valid ICO; qed"));
-            tray_icon
-                .with_tooltip("Space Acres")
-                .with_menu(Menu::new([
-                    MenuItem::button(T.tray_icon_open(), TrayMenuSignal::Open),
-                    MenuItem::button(T.tray_icon_close(), TrayMenuSignal::Close),
-                ]))
-                .build({
-                    let sender = sender.clone();
-                    move |tray_event| {
-                        if let TrayEvent::Menu(signal) = tray_event {
-                            match signal {
-                                TrayMenuSignal::Open => sender.input(AppInput::ShowWindow),
-                                TrayMenuSignal::Close => sender.input(AppInput::ShutDown),
-                            }
-                        }
-                    }
-                })
-                .map_err(|error| {
-                    warn!(%error, "Unable to create tray icon");
-                })
-                .ok()
-        };
+        let tray_icon = TrayIconBuilder::new();
+
+        let image = image::load_from_memory(ICON)
+            .expect("Statically correct image; qed")
+            .to_rgba8();
+
+        let (width, height) = image.dimensions();
```

Comment:
These variables will be unused on Windows and cause clippy warnings. They should be grouped with tray icon construction using `let tray_icon = {};`.

Also you can use `image::load_from_memory_with_format(ICON, image::ImageFormat::Png)` as more concrete method and you can use `image.width()` instead of extracting separate variables first.